tests/integration/test_authd/test_authd_use_source_ip.py

```diff
@@ -0,0 +1,160 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: This module verifies the correct behavior of the setting 'use_source_ip'.
+
+components:
+    - authd
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-authd
+    - wazuh-db
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+tags:
+    - enrollment
+'''
+import os
+import ssl
+import time
+import pytest
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_PATH
+from wazuh_testing.utils.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.modules.authd.utils import validate_authd_response
+
+from . import CONFIGURATIONS_FOLDER_PATH, TEST_CASES_FOLDER_PATH
+
+# Marks
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+
+# Configurations
+test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'config_authd_use_source_ip.yaml')
+test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'cases_authd_use_source_ip.yaml')
+test_configuration, test_metadata, test_cases_ids = get_test_cases_data(test_cases_path)
+test_configuration = load_configuration_template(test_configuration_path, test_configuration, test_metadata)
+
+# Variables
+
+log_monitor_paths = []
+monitored_sockets_params = [('wazuh-modulesd', None, True), ('wazuh-db', None, True), ('wazuh-authd', None, True)]
+receiver_sockets, monitored_sockets, log_monitors = None, None, None  # Set in the fixtures
+daemons_handler_configuration = {'all_daemons': True, 'ignore_errors': True}
+
+# Fixtures
+@pytest.fixture(scope='function')
```

Comment:
```suggestion
@pytest.fixture()
```
Also, move to `conftest.py` file from auth tests.